src/diagnostics/AudioTest.js

```diff
@@ -20,7 +20,9 @@ class AudioTest extends Test {
       this.reject(new Error('audio timeout'));
     }, this.volumeTimeout);
 
-    this.localMedia.start(this.options, (err) => {
+    const options = Object.assign({}, this.options, { video: false });
```

Comment:
this fixes an issue where audio test would fail if you don't have camera permissions or don't have a camera.